homeassistant/components/mqtt/__init__.py

```diff
@@ -1247,6 +1265,18 @@ def device_info(self):
         return device_info_from_config(self._device_config)
 
 
+@websocket_api.websocket_command(
+    {vol.Required("type"): "mqtt/device/remove", vol.Required("device_id"): str}
```

Comment:
There's nothing specific to MQTT in the implementation, should this maybe be moved elsewhere?